### PR TITLE
fix: remove component variables for sftable

### LINKED ISSUE #1554: [BUG] Incoherent variable naming in SfTable
**Describe the bug**
https://github.com/DivanteLtd/storefront-ui/blob/develop/packages/shared/styles/components/organisms/SfTable.scss

Variables:
```
--table-border-heading
--table-border-heading-width
```
propably should be have word order like other variables:
```
--table-heading-border
--table-heading-border-width
```

Also `--_table-column-order` should be without underscore (typo).

If this issue will be accepted You could **assign me** for this.

**How to reproduce**
Visible in:
https://github.com/DivanteLtd/storefront-ui/blob/develop/packages/shared/styles/components/organisms/SfTable.scss
and the docs:
https://docs.storefrontui.io/components/table.html#css-custom-properties-variables

### PR DESCRIPTION
**Pull request by mayashavin**
# Related issue
Closes #1554

# Scope of work
- Remove component specific CSS Variables, only left the following:
 * --table-column-flex
 * --table-row-box-shadow
 * --table-column-width

# Screenshots of visual changes

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed
- [x] Changes documented 

If applicable:

- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/shared/styles/components/organisms/SfTable.scss [hunk 1]
```diff
@@ -12,59 +12,46 @@
...
-    --table-row-border-width: 0;
+    .sf-table__row {
+      border-width: 0;
+    }
   }
   @include for-desktop {
-    --table-row-padding: var(--spacer-sm) var(--spacer-sm);
-    --table-heading-padding: var(--spacer-sm) var(--spacer-sm);
-    --table-heading-background: transparent;
     --table-column-flex: 1;
```
Comment (filrak) [inline packages/shared/styles/components/organisms/SfTable.scss:53]: why you left this one + 2 others?

**Review by filrak — COMMENTED** [review]
Shouldn't we also remove the declarations?

File: packages/shared/styles/components/organisms/SfTable.scss (hunk 1 above)
Comment (mayashavin) [reply packages/shared/styles/components/organisms/SfTable.scss:53]: this is for the column no? I'm just not so sure we should remove it. @filrak

**Review by justyna-13 — COMMENTED** [review]
Great! :+1:  When PR with releases will be merged it'd be good to add also information in docs about removing these variables in SfTable :)

**Comment by mayashavin** [comment]
@justyna-13 I think we will add those in the release note. LMK otherwise

**Comment by rain2o** [comment]
@mayashavin PR submitted. Not sure if that's what you had in mind regarding the removal of the component variables.

**Review by Baroshem — CHANGES_REQUESTED** [review]
I do not have any comments about the code itself, the change looks good. However there are many differences between develop sftable-remove-scss branches. This needs to be sorted out first. Also, SfTable on this branch looks worse than on develop branch (may be due to the change of column order (please see attached screenshot)

`develop`
Screenshot 2021-03-03 at 08 30 15

`sftable-remove-scss`
Screenshot 2021-03-03 at 08 34 37

**Pull request closed without merging**